components/Piano/Piano.tsx

```diff
@@ -96,6 +96,8 @@ const _Piano: FunctionComponent<PianoProps> = ({
             onMouseUp={() => onMouseUp(midi)}
             onMouseEnter={isMousePressed ? () => play(midi) : undefined}
             onMouseLeave={() => stop(midi)}
+            onTouchStart={()=>onMouseDown(midi)}
+            onTouchEnd={()=>onMouseUp(midi)}
```

Comment:
I guess you pushed this by disabling husky ? That's why [prettier](https://prettier.io/) didn't run automatically when you committed the code. Can you please run `yarn format` and push the changed files? That will fix the formatting here.

Reply:
Sure i will do it